srcs/Client.cpp

```diff
@@ -61,15 +61,48 @@ void Client::recvStartLine(const std::string &line)
  */
 void Client::recvHeader(const std::string &line)
 {
-	// TODO: \n 단위로 파싱을 해서 줄별로 넘겨받습니다.
-	//       각 라인을 파싱해서 map에 넣어주시면 됩니다.
-	std::cout << "HEADER : " << line << std::endl;
+	// std::cout << "HEADER : " << line << std::endl;
+	std::vector<std::string> headers = ft_split(line, ':');
+	this->_request.insertToHeader(headers[0], line.substr(headers[0].size() + 1));
```

Comment:
http 헤더의 `key: value` 쌍에서 `:` 뒤에 공백이 하나 있기 때문에 한 자리 더 옮겨서 substr을 따주는 것이 맞는 것 같습니다!

```suggestion
	this->_request.insertToHeader(headers[0], line.substr(headers[0].size() + 2));
```

Reply:
<수정 사항>
`field-name ":" OWS field-value OWS` 이므로 whitespace를 optional하게 처리하도록 로직을 변경해야 할 것 같습니다.